components/Tabs/tests/unit/Theming/TabsThemingTests.m

```diff
@@ -0,0 +1,191 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
Copyright notice to 2019.

Reply:
Addressed in 82f41bedaeaae0376513d405303d6d8b4f320c34